### PR TITLE
Update German (de) Staking, Learn 2, and Contributing buckets

### PR DESCRIPTION
**Pull request by minimalsm**
## Description

Update German (de) buckets: 

- Staking
- Learn 2
- Contributing

### CODE REVIEW

File: src/content/translations/de/enterprise/index.md [hunk 1]
```diff
@@ -76,10 +76,10 @@ Verschiedene Organisationen unternahmen gemeinsame Anstrengungen, um Ethereum un
 
 Die Erstellung einer Anwendung auf einer Layer-2-Skalierbarkeitslösung [kann helfen, viele der Bedenken, die Unternehmen früher dazu gebracht haben, auf privaten Blockchains aufzubauen, zu beseitigen](https://entethalliance.org/how-ethereum-layer-2-scaling-solutions-address-barriers-to-enterprises-building-on-mainnet/), behält jedoch die Vorteile des Aufbaus auf dem Mainnet.
 
-Beispiele für L2-Lösungen, die produktionsbereit sind oder bald sein werden:
+Beispiele für L2-Lösungen, die produktionsbereit sind oder es bald sein werden:
 
 - Optimistic Rollups (Daten on-chain, Betrugsnachweise)
-  - [Optimism](https://optimism.io/)
+  - [Optimismus](https://optimism.io/)
```
Comment (corwintines) [inline src/content/translations/de/enterprise/index.md:82]:
```suggestion
  - [Optimism](https://optimism.io/)
```

I dont think this is supposed to be translated

Comment (lukassim) [reply src/content/translations/de/enterprise/index.md:82]: Exactly, good catch @corwintines.

File: src/content/translations/de/staking/solo/index.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+alt: Leslie das Nashorn auf ihrem eigenen Computerchip.
+summaryPoints:
+  - Erhalten Sie maximale Belohnungen direkt vom Protokoll, indem Sie sicherstellen, dass Ihr Validator ordnungsgemäß funktioniert und online ist
+  - Benutzen Sie Hardware zu Hause und tragen Sie persönlich zur Sicherheit und Dezentralisierung des Ethereum-Netzwerks bei
+  - Vertrauen Sie niemandem und geben Sie niemals den Zugang zu Ihren Geldern weiter
+---
+
+## Was ist Solo-Staking? {#what-is-solo-staking}
+
+Solo-Staking ist das [Betreiben eines Ethereum-Knotens](/run-a-node/), der mit dem Internet verbunden ist, und das Hinterlegen von 32 ETH, um einen
```
Comment (corwintines) [inline src/content/translations/de/staking/solo/index.md:17]: Something wonky with </a> tag here. Will cicle back to get in on Monday

File: src/content/translations/de/staking/solo/index.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+Validator< zu aktivieren /a>, wodurch Sie direkt am Netzwerkkonsens teilnehmen können.
```
Comment (minimalsm) [inline src/content/translations/de/staking/solo/index.md:18]:
```suggestion
Solo-Staking ist das [Betreiben eines Ethereum-Knotens](/run-a-node/), der mit dem Internet verbunden ist, und das Hinterlegen von 32 ETH, um einen Validator zu aktivieren, wodurch Sie direkt am Netzwerkkonsens teilnehmen können.
```

**Review by corwintines — APPROVED** [review]
Looks like this timed out, but was working locally so going to approve and merge. Thanks @minimalsm!

**Pull request merged**